developer-guides/realtime-api/the-message-object/README.md

```diff
@@ -40,32 +42,146 @@ The attachment object can contain several fields:
 {
     "messages": [
         {
-            "_id": "message-id",
-            "rid": "room-id",
+            "_id": "<message-id>",
+            "rid": "<room-id>",
             "msg": "Hello World!",
             "ts": { "$date": 1480377601 },
             "u": {
-                "_id": "user-id",
-                "username": "username"
+                "_id": "<user-id>",
+                "username": "<username>"
             },
             "_updatedAt": { "$date": 1480377601 }
         },
         {
-            "_id": "message-id",
-            "rid": "room-id",
+            "_id": "<message-id>",
+            "rid": "<room-id>",
             "msg": "Hello!",
             "ts": { "$date": 1480377601 },
             "u": {
-                "_id": "user-id",
-                "username": "username"
+                "_id": "<user-id>",
+                "username": "<username>"
             },
             "_updatedAt": { "$date":1480377601 },
             "editedAt": { "$date": 1480377601 },
             "editedBy": {
-                "_id": "user-id",
-                "username": "username"
+                "_id": "<user-id>",
+                "username": "<username>"
             }
         }
     ]
 }
 ```
+## Payload
+
+Depending on the ```type```, the payload can include "audio", "video", "image", "file", etc.. or more complex rich messages using [templates](templates/).
+
+The fields inside the payload object depend on the type and can contain any information needed to render the message. 
+
+When using templates the payload object must include this field:
+
+- `template_type`: Which type of template is to be used when parsing and rendering the rich message content.  A few specific template_types are described [here](templates/)
```

Comment:
I think you will describe this in the template page, right? No need to also mention it here.